codebase/codebsae_test.go

```diff
@@ -0,0 +1,81 @@
+package codebase_test
```

Comment:
my bad.
Thank you, Fixed in https://github.com/almighty/almighty-core/pull/907/commits/8de25a0a2894a9f544f59404ba553db520d79ca3